src/app/components/list/toolbar/index.html

```diff
@@ -128,13 +128,44 @@ <h3>
   </sky-demo-page-example>
 	
 	<sky-demo-page-content sectionHeading="UX guidelines">
-		<div class="sky-subsection-heading">Sorting</div>
-		<p>If you use the toolbar with a grid view, you have two options for letting users sort the list:</p>
-		<ul>
-			<li>The sort button in the toolbar</li>
-			<li>Clickable column headers</li>
-		</ul>
-		<p>Don't combine both sorting options in a single list because it confuses users about which option is currently applied.</p>
+		
+		<section>
+			<header><div class="sky-section-heading">Usage</div></header>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>
+						The list toolbar appears above a <a routerLink="/components/list">list</a> of records. It contains actions to manipulate the contents and display of the list. Place frequently used actions directly in the toolbar, and place less-frequently used actions in a more actions menu. When multiple views of the list are available, the view switcher is docked on the right side of the toolbar.
+					</p>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/list-toolbar-context.png" style="width: 100%;" /></div>
+				</stache-column>
+			</stache-row>
+		</section>
+		
+		<section>
+			<header><div class="sky-section-heading">Composition</div></header>
+			<div class="screenshot screenshot-info"><img src="~/assets/img/guidelines/toolbar/anatomy.png" style="width: 100%;" /></div>
+			<p>For more information about how to arrange the content in a list toolbar see the <a routerLink="/components/toolbar">toolbar</a> UX guidelines.</p>
+		</section>
+		
+		<section>
+			<div class="sky-section-heading">Interactions</div>
+			<div class="sky-subsection-heading">Sorting</div>
+			<p>If you use the toolbar with a grid view, you have two options for letting users sort the list:</p>
+			<ul>
+				<li>The sort button in the toolbar</li>
+				<li>Clickable column headers</li>
+			</ul>
+			<p>Don't combine both sorting options in a single list because it confuses users about which option is currently applied.</p>
```

Comment:
I'd change "combine" to "use" and change "a single" to "the same." And I'd also change "confuses" to "can confuse" and delete "currently."

Reply:
ok